requirements/libraries.txt

```diff
@@ -19,14 +19,15 @@ govdelivery==1.2
 Jinja2==2.10.1
 jsonfield==2.0.2
 lxml==4.2.5
-Markdown==2.6.11
-markdown-emdash==0.1.0
```

Comment:
Is `markdown-emdash` no longer needed or is it rolled into the regdown package as a dependency there?

Reply:
Very good question! The existing markdown-emdash doesn't appear very actively maintained and uses a deprecated Python-Markdown API, so I added an alternative, Markdown 3.1-comptaible alternative directly to Regdown.

This does make my realize though that for cfgov-refresh we should pin the Markdown requirement.